ch04.adoc

```diff
@@ -202,312 +218,216 @@ Optionally, a time coordinate variable may be indicated additionally by providin
+In the **Months** column, "usual" means there are 12 months with the usual lengths expected by calendar software (31 days in January, etc.).
+These month lengths are the same in all calendars marked "usual", except that the calendars differ regarding the rules for determining leap years, when February has 29 days instead of 28.
+In the **Leap days** column, "Gregorian" and "Julian" identify different rules for determing which years are leap years.
+By the Gregorian rule, a year is a leap year if either (i) it is divisible by 4 but not by 100 or (ii) it is divisible by 400.
+By the Julian rule, any year that is divisible by 4 is a leap year, even if it is also divisible by 100.
+In the **Year 0** column, "yes" and "no" respectively indicate that there are, or are not, valid dates in year 0 in this calendar.
+In calendars marked with *, dates in year 0 are climatological; this use of year 0 is deprecated (see <<climatological-statistics>>).
+As well as those listed in this table, the **`calendar`** attribute may also take the value **`none`** or any other value, with meanings given at the end of <<calendar>>.
+
+In a given calendar, each valid datetime identifies a particular instant in the continuous physical dimension of time.
+The reference datetime in the **`units`** of a time coordinate variable must be a valid datetime in the calendar of the variable, and identifies a particular instant according to that calendar.
+A datetime which is invalid in a given calendar cannot be converted into a time-coordinate in that calendar.
+For example, 2025-01-31 00:00 is a valid datetime in the **`standard`** calendar but not in the **`360_day`** calendar (in which January has 30 days), while 2025-02-29 11:00 is valid in the **`all_leap`** calendar (in which 2025 is a leap year) but not in the **`standard`** calendar.
+
+A given instant in the continuous physical dimension of time may be identified by different datetimes in different calendars.
+For example, 1917-11-07 12:00 in the **`standard`** calendar and 1917-10-25 12:00 in the **`julian`** calendar identify the same instant.
+Because the calendars have different sets of valid datetimes, a given time coordinate value with given reference datetime in its **`units`** can represent different datetimes in different calendars.
+For example, a time coordinate of **1** with **`units="days since 2020-02-28 23:10"`** represents the datetime 2020-02-29 23:10 in the **`standard`** calendar, but 2020-03-01 23:10 in the **`no_leap`** calendar (in which 2020 is not a leap year).
+
+In the **`julian`** and the **`standard`** calendar, dates in years before year 0 (i.e. before 0-01-01 00:00:00) are not allowed, and the year in the reference datetime of the units must not be negative.
+In these calendars, year zero has a special use to indicate a climatology (see <<climatological-statistics>>), but this use of year zero is deprecated.
```

Comment:
This is an exact (at least very nearly) duplicate of what was stated just two paragraphs up: 
Delete, or shorten substantially and mergeinto next sentence.

Reply:
I know this is an ancient recommendation. I've changed it to "In these calendars, datetimes in year 0 indicate a climatology. This use is deprecated (see `<<climatological-statistics>>` for the recommended mechanism)."